### ISSUE TITLE
How to install it?

### ISSUE THREAD

**Issue by michaldybczak** [ajoshpratt/kwinOverview#2]
Instructions provided here lead to nowhere. I cd'ed to Dowloads and did:

```
git clone https://github.com/ajoshpratt/kwinOverview.git
cd kwinOverview
plasmapkg --type kwinscript -i .
```
I got output that it was successfully recorded.

and... now what? I don't see it on kwin scripts settings, shortcuts are a massive and complex window so I would need more pointers where I could find it (if it's there) or maybe create custom shortcut, if so how? At the moment shortcuts settings are usable but overwhelming and often confusing, because they require a priori knowlege where to find proper option/shortcut or how it is named (with localisation it gets even harder to track down the right element).

At the moment I looked under: global shortcuts KWin, but didn't find anything.

There is a way to install kwin scripts manually from a package but for that we need .kwinscript file but there is no such file here, so that's again a no go.

Basically, an average non-tech user cannot install it, at least not with current instructions.
Is there a chance to put it on kde.store so we could simply download/install script the easy and user friendly way?

Thanks

**Comment by j605** [comment]
This didn't work for me but the installation instructions given in https://github.com/rliang/kwin-dynamic-virtual-desktops works. Just adapt those commands :)

**Comment by ajoshpratt** [comment]
@michaldybczak, did @j605's comment work?  @j605, thanks for those.

I'll look into making a kwinscript for this so that it can be 'properly' installed.  It's interesting that it doesn't work; have you restarted KWin/Plasma since installing?  If so, what distro are you using?

**Comment by michaldybczak** [comment]
No, I didn't restart kwin or Plasma. I use Manjaro KDE. Will also check out those instructions that @j605 posted.
Thanks guys!

**Comment by JMCavanillas** [comment]
On Solus (Plasma 5.12.5), I managed to install it using **plasmapkg2** instead of plasmapkg. @j605 instructions didn't work. There is a kpluginindex.json file I suppose you must modify too in order to do it in that way

**Comment by astatide** [comment]
@JMCavanillas, I didn't even notice I'd left the 2 off.  Yeah, plasmapkg no longer works, ha!  plasmapkg2 is what Plasma 5 uses.

**Comment by michaldybczak** [comment]
I confirm:

`kwinOverview]$ plasmapkg2 --type kwinscript -i .`

make the script appear on the list without reloading. Now to find out how to activate it.

**Comment by michaldybczak** [comment]
Since this issue is resolved, I'm closing the topic.
